packaging/nuget/tools/init.ps1

```diff
@@ -33,14 +33,19 @@ try {
 
 	Set-ItemProperty -Path $platformKeyPath -Name "NuGetUser" -Value "true" | Out-Null
 
-
-    $url = "http://particular.net/download-the-particular-service-platform?version=$packageVersion" 
-    $url = $url.ToLowerInvariant(); 
-
-    if($dte){
-	    $dte.ExecuteCommand("View.URL", $url)
-    }
+	Write-Verbose 'Reporting first time install and version information to www.particular.net. This call does not collect any personal information. For more details, see the License Agreement and the Privacy Policy available here: http://particular.net/licenseagreement. Subsequent NuGet installs or updates will not invoke this call.' -verbose
+	$url = 'https://particular.net/api/ReportFirstTimeUsage'
+	$postData  = New-Object System.Collections.Specialized.NameValueCollection
+	$postData.Add("version", $packageversion)
+	$wc = New-Object System.Net.WebClient
+	$wc.UseDefaultCredentials = $true
+	$wc.UploadValuesAsync($url,"post", $postdata)
 } 
 Catch [Exception] { 
 	Write-Warning $error[0]
 }
+finally {
+	if ($wc){
+	$wc.Dispose()
```

Comment:
I think what @danielmarbach is getting at is since the powershell script is never `await`ing the result of the async `UploadValuesAsync` call, that means the finally block could be executed and dispose the WebClient before the call finishes.